### DISCUSSION TITLE
Installation docs refer to a non-existent docker image tag

### DISCUSSION THREAD

**Discussion by sparrowt** [aquasecurity/trivy#8145] [Documentation]
### Description

https://trivy.dev/latest/getting-started/installation/#container-image-official says:
> Example:
> `docker run -v /var/run/docker.sock:/var/run/docker.sock -v $HOME/Library/Caches:/root/.cache/ aquasec/trivy:0.58.0-1-g775f954c3 image python:3.4-alpine`

However that image tag doesn't exist:
https://hub.docker.com/r/aquasec/trivy/tags?name=0.58.0-1-g775f954c3

so docker fails with this:
> docker: Error response from daemon: manifest for aquasec/trivy:0.58.0-1-g775f954c3 not found: manifest unknown: manifest unknown.

The documentation source is [here](https://github.com/aquasecurity/trivy/blob/main/docs/getting-started/installation.md#container-image-official) and doesn't have a hard-coded tag it uses `{{ git.tag[1:] }}` but I haven't yet figured out where this is provided from.

Either way for some reason it is providing a tag which does not exist on Docker Hub.

### Link

https://trivy.dev/latest/getting-started/installation/#container-image-official

### Suggestions

Example needs fixing to reference a real tag, or just `latest` I suppose

**Comment by nikpivkin** [comment]
It seems to be related to manually running the site deployment workflow. 

/cc @itaysk

**Reply by knqyf263** [reply]
And v0.58 and v0.58.0 exist now.
![CleanShot 2024-12-24 at 08 52 21](https://github.com/user-attachments/assets/e9924f5f-9e2b-47c7-98ee-13487815ba32)

**Reply by itaysk** [reply]
not sure yet if/how it relates to the manual push, but I'll first remove those versions

**Reply by itaysk** [reply]
FTR, I just re-run the manual deployment and it didn't create those versions

**Comment by itaysk** [comment]
thanks, I've created https://github.com/aquasecurity/trivy/issues/8182
